clients/geth/specular/data/ruleset.js

```diff
@@ -0,0 +1,22 @@
+// rules which govern clef key management behavior
+// for more info, see: https://geth.ethereum.org/docs/tools/clef/rules
+
+function ApproveTx(r) {
+  // all transactions not going to the rollup addr are rejected
+  // TODO: do we need additional / more fine grained rules?
+
+  // this file can't access env variables so addresses have to be hardcoded
```

Comment:
cant we just run `configure_system.sh` from within this script and then read the env variable